### ISSUE TITLE
Navbar right issue

### ISSUE THREAD

**Issue by Namennaj** [geedmo/yamm3#56]
When using navbar right the dropdowns are not behaving as expected.  Look at this fiddle:
http://jsfiddle.net/7oLwma1t/

Click dropdown 1

**Comment by jmuheim** [comment]
This indeed is a problem. I face it too.

I managed to work around it by removing `position: static` from `.yamm .nav`, but this made my yamm element collapse.

Before:

![image](https://cloud.githubusercontent.com/assets/1814983/22943137/b114f3f6-f2ec-11e6-949c-724a595496be.png)

After:

![image](https://cloud.githubusercontent.com/assets/1814983/22943161/c217615c-f2ec-11e6-91f6-e7ffdb1e67d2.png)

I fixed this by adding `whitespace: nowrap` to my two input fields (`.login_and_password { white-space: nowrap }`)

Still it would be good to have a fix for this problem.
